app/presenters/region_presenter.rb

```diff
@@ -58,18 +58,26 @@ def total_polygons_percentage
   def geometry_ratio
     polygons_count = 0
     points_count = 0
-    
+
     @countries.each do |country|
       statistic = country.statistic
-      polygons_count += (statistic && statistic.polygons_count) || 0
-      points_count += (statistic && statistic.points_count) || 0
+      polygons_count += statistic&.polygons_count || 0
+      points_count += statistic&.points_count || 0
     end
-    
+
     total = polygons_count + points_count
-    
+
     {
-      polygons: (((polygons_count/total.to_f)*100).round rescue 0),
-      points:   (((points_count/total.to_f)*100).round   rescue 0),
+      polygons: begin
+                  ((polygons_count / total.to_f) * 100).round
+                rescue StandardError
+                  0
+                end,
+      points: begin
+                ((points_count / total.to_f) * 100).round
+              rescue StandardError
+                0
+              end
```

Comment:
Any particular reason for adding the entire begin-rescue blocks here?
I personally find this less readable while achieving the same thing.

Potentially this could be refactored to not use rescue at all and check the value of the variables before doing calculations